data/cameras.xml

```diff
@@ -491,7 +521,7 @@
 			<Hint name="invert_sraw_wb" value=""/>
 		</Hints>
 	</Camera>
-	<Camera make="Canon" model="Canon EOS 5DS R" decoder_version="6">
+	<Camera make="Canon" model="Canon EOS 5DS R" decoder_version="6" supported="no">
```

Comment:
The rawspeed support does seem fine but since there are no final images we didn't want to enable it yet. I guess that's a darktable thing, I'll remove it from the commit.